### DISCUSSION TITLE
为什么我PCL2糊了

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5282] [提问] [answered]
怎么从笔记本的屏幕拖到外接的显示器上就糊了，是Windows显示设置缩放的锅吗）

![image](https://github.com/user-attachments/assets/0543d823-33f9-483a-b63d-33aeaf1ac7d9)

![image](https://github.com/user-attachments/assets/28734da5-1d8b-4a8b-97a5-d13ec4315bf2)

**Comment by Pigeon0v0** [comment]
是的

**Reply by E0x72-21** [reply]
那有解决方法吗）

**Comment by MoYuan-CN** [comment]
是 Windows 分辨率与显示器原生分辨率差异导致的
大致可以参考下面这篇文章解决？
- https://www.sysgeek.cn/windows-11-fix-blurry-screen/

**Reply by E0x72-21** [reply]
同缩放大小当然就没这毛病了，但正是因为屏幕大小(物理尺寸大小)的区别我才要不同的缩放比例
顺带一提，PCL2能否做成像cdi和cdm一样检测到窗口跨屏幕就自动适应缩放？

https://github.com/user-attachments/assets/564a5855-bafb-4a24-9791-4d601a93dd86

**Comment by Silverteal** [comment]
估计是高 DPI 缩放问题，查找到以下相关开发信息：
[支持 Windows 10 最新 PerMonitorV2 特性的 WPF 多屏高 DPI 应用开发](https://blog.walterlv.com/post/windows-high-dpi-development-for-wpf.html)

---

@E0x72-21 
您有空时可尝试一下此版本：https://github.com/Hex-Dragon/PCL2/actions/runs/12510355571/artifacts/2364642254

**Reply by E0x72-21** [reply]
这版本好使，顺带问问这功能会在日后合并到正式版吗？

**Reply by MoYuan-CN** [reply, marked as answer]
- #5337
